SeaPublicWebsite/ExternalServices/EmailSending/GovUkNotifyConfiguration.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SeaPublicWebsite.ExternalServices.EmailSending
+{
+    public class GovUkNotifyConfiguration
+    {
+        public const string Name = "GovUkNotify";
+        public string Test { get; set; }
+        public ApplicationReferenceNumberConfiguration ApplicationReferenceNumber { get; set; }
```

Comment:
It seems a bit unnecessary to have three separate files for this config object. As the classes are only for storing information I think it's fine to make `ApplicationReferenceNumberConfiguration` and `RequestDocumentConfiguration` inner classes in this file.